yarn.lock

```diff
@@ -1905,6 +1905,40 @@ babel-plugin-polyfill-regenerator@^0.1.2:
   dependencies:
```

Comment:
There are yarn.lock and package-lock.json.
Why both are added? are both synchronised? 
It is better to have only one.